Library/Homebrew/cmd/bottle.rb

```diff
@@ -95,10 +95,10 @@ def print_filename(string, filename)
       end
     end
 
-    keg_contains_absolute_symlink_starting_with?(string, keg) || result
+    keg_contains_absolute_symlink_starting_with(string, keg) || result
```

Comment:
I think we want to retain this order, too. We want the side effect of output in verbose mode, if there are any offending symlinks.